src/components/header/Header.scss

```diff
@@ -130,7 +136,7 @@
 }
 
 .header .menu-btn:checked ~ .menu-icon .navicon {
-  background: transparent;
+  background: transparent !important;
```

Comment:
Any particular reason to set this as important?

Reply:
I did that to fix the menu icon button in dark mode

![IMG_20210624_062104.jpg](https://user-images.githubusercontent.com/80502737/123185840-72ca6280-d4b4-11eb-8ebd-599582903f23.jpg)